code/modules/clothing/spacesuits/void/void.dm

```diff
@@ -241,5 +265,16 @@
 			W.forceMove(src)
 			tank = W
 		return
+	else if(istype(W,/obj/item/device/suit_cooling_unit))
+		if(cooler)
+			user << "\The [src] already has a suit cooling unit installed."
+		if(tank)
+			user << "\The [src]'s airtank is in the way.  Remove it first."
```

Comment:
This should be `else if(tank)`, otherwise you'll be able to install a cooler when you already have one installed. Ditto above with tanks.